.golangci.yml

```diff
@@ -31,6 +31,10 @@ linters:
     - unparam
     - unused
 linters-settings:
+  gosec:
+    # TODO - remove this and fix linter complaint
+    excludes:
+      - G115
```

Comment:
How many were there? It would be a little better to mark them directly as `//nolint:gosec // G115 ....` if it's not too much trouble.

Reply:
10+